teachopencadd/talktorials/T038_protein_ligand_interaction_prediction/talktorial.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<ul><li>read in -&gt; read</li><li>process it / process them (and data -&gt; data points)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>